iFixit/react-commerce
# test: Add filters test

This PR adds filters test according to #124 

Tests have been implemented following [guiding principles](https://testing-library.com/docs/guiding-principles) of Testing Library, and this led also to accessibility improvements (this is why there are a few changes in the filters section)

## Changelog

- Update dependencies
- Refactor newsletter tests
- Add product list filters test

## QA

This is mostly a PR that adds tests. These tests are run on CI, so if they pass it means that filters are still working. 
We might want to do a smoke test to verify that everything else still works.
